app/src/main/res/layout/fragment_correction.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".view.quiz.CorrectionFragment"
+   >
+
```

Comment:
I don't know if spotless did that, but you should check the spacing in this file, because it is a mess, let's be honest ;).

Reply:
Spotless does things, terrible things

Reply:
Ah I don't know what was the benefit of using spotless, looking back at our project ;)